spec/requests/api/v0.1/info_spec.rb

```diff
@@ -3,11 +3,13 @@
 require "rails_helper"
 
 describe "info endpoint" do
-  it_behaves_like "a register API endpoint", "/api/v0.1/info"
+  let!(:auth_token) { create(:authentication_token, hashed_token: "valid_token") }
+
+  it_behaves_like "a register API endpoint", "/api/v0.1/info", "valid_token"
 
   context "with a valid authentication token and the feature flag on" do
     it "returns status 200 with a valid JSON response" do
-      get "/api/v0.1/info", headers: { Authorization: "Bearer bat" }
+      get "/api/v0.1/info", headers: { Authorization: "valid_token" }
```

Comment:
Shouldn't it be more like

```suggestion
      get "/api/v0.1/info", headers: { Authorization: auth_token }
```